src/Compilers/CSharp/Portable/Symbols/TypeSymbolWithAnnotations.cs

```diff
@@ -324,25 +224,6 @@ private TypeSymbolWithAnnotations(TypeSymbol defaultType, NullableAnnotation nul
             _extensions = extensions;
         }
 
-        public TypeSymbolWithAnnotations AsSpeakable()
```

Comment:
>AsSpeakable [](start = 41, length = 11)

note: there is a mention of "unspeakable" in the design doc. Could you remove it too? Thanks